pipelines/dependency-builds.yml.erb

```diff
@@ -292,13 +283,11 @@ version_lines = dep['buildpacks'].values.reduce([]) {|sum, bp| sum | get_version
           - name: all-monitored-deps
   <% end %>
   <% if dep_name.downcase == 'node' && line.downcase == 'node-lts' %>
-    - task: create-tracker-story
+    - task: create-new-version-line-story-node
```

Comment:
why Node specifically?

Reply:
just trying it there first?

Reply:
Not trying out. This is a special task exclusively for node similar to the task on line 35 for all other dependencies.
In my understanding of the file's history, Node is considered special because its LTS dependency alone is needed in Ruby and .NET buildpacks and is accomplished by maintaining a special dependency named `node-lts` in depwatcher. Looks like @brayanhenao implemented this in #130.
In this PR, I haven't looked deeply into rearchitecting the pipeline and just tried to replace tracker with github.